doc/library/tensor/basic.txt

```diff
@@ -427,7 +427,8 @@ TensorVariable
     you'll want to call.
 
 
-.. class:: _tensor_py_operators(object)
+.. autoclass:: _tensor_py_operators
+   :members:
```

Comment:
Why this change? I don't see a change in the doc generated. I don't have objection to this change.